### DISCUSSION TITLE
Google auth with Flutter web Example

### DISCUSSION THREAD

**Discussion by nomanmurtaza786** [supabase/supabase#2294] [Questions] [answered]
Hi there, 
Any example of using Flutter web (not iOS and Android ) with Google auth?

**Comment by dshukertjr** [comment, marked as answer]
Hi @nomanmurtaza786 !

Here is an example of using Google Auth with Flutter! I just tried it on Flutter web, and was able to login successfully! 

https://github.com/dshukertjr/supabase-flutter-demo/tree/feat/google-auth

**Reply by basicBrogrammer** [reply]
When I try to use google auth on the iOS simulator it launches the webview browser, but it doesn't close on success. Is this a bug in the supabase-flutter code? 

The authentication is successful. The webview browser just stays in a "loading" state.

**Reply by dshukertjr** [reply]
@basicBrogrammer Do you mind opening a separate question with detailed procedures of how to reproduce your situation?
